seissolxdmfwriter/README.md

```diff
@@ -30,5 +30,25 @@ sxw.write(
     reduce_precision=True,
     backend="hdf5",
 )
+
+# Finally, the module can be use to write data directly from seissolxdmf, limiting
+# the memory requirements
+
+sxw.write_from_seissol_output(
+    'test-fault-sx',
+    fn,
+    ['SRs', 'SRd','fault-tag', 'partition'],
+    [3,4],
+    reduce_precision=True,
+    backend="hdf5",
+    compression_level=4,
+)
+
 ```
 
+The module also encapsulates `seissol_output_extractor`, which can be used to extract and process data from SeisSol output files, allowing selection of variables, time steps, spatial ranges, and output format.
+Here is an example of use:
+
+```bash
+seissol_output_extractor test-fault.xdmf --time "i2,i4" --variable PSR Vr partition
```

Comment:
Can you write a bit more about the features? E.g. how do  I extract spatial ranges, what's the `i2,i4` stand for ...